07-Driving compliance via Bugs/Readme.md

```diff
@@ -169,6 +169,10 @@ Any bug that has been resolved before can be reactivated if the control failure
 ```
 After setting any one of the above policies, commit the file and add an entry for *ControlSettings.json* in *ServerConfigMetadata.json* (in the repo).
 
+## Customizing bug logging for certain controls
+
+Bug only for specific list of controls can be controled by the organization policy using "CustomControlList". Add controls in the CustomControlList list and use "-AutoBugLog Custom" option in command parameter. It will run bug logging only for the controls which are added in the CustomControlList.
```

Comment:
controled --> controlled

Bug logging can be restricted to specific controls using  "CustomControlList" option in org policy . Add list of controls in the CustomControlList parameter and use "-AutoBugLog Custom" option in command parameter. It will run bug logging only for the controls which are added in the CustomControlList.